src/Tribe/Admin/Move_Tickets.php

```diff
@@ -662,18 +657,15 @@ public function move_tickets( array $ticket_ids, $tgt_ticket_type_id, $src_event
 			update_post_meta( $ticket_id, $ticket_type_key, $tgt_ticket_type_id );
 			update_post_meta( $ticket_id, $ticket_event_key, $tgt_event_id );
 
-			// adjust sales numbers - don't allow negatives
-			$src_qty_sold --;
-			$tgt_qty_sold ++;
-			update_post_meta( $src_ticket_type_id, 'total_sales', $src_qty_sold );
-			update_post_meta( $tgt_ticket_type_id, 'total_sales', $tgt_qty_sold );
-
-			//adjust stock numbers for RSVP Tickets
+			// adjust stock numbers for RSVP and Commerce Tickets.
 			if ( 'Tribe__Tickets__RSVP' === $ticket['provider'] ) {
```

Comment:
```suggestion
			if ( 'Tribe__Tickets__RSVP' === $ticket['provider'] ) {
```
Looking at this function I'm wondering if it is time to refactor so it isn't so big and logic heavy. We are about 200 lines long so far.

Are there points of diverging logic like this check, that we can separate into `do_rsvp_stuff()`. Don't force it but if you see some quick wins here that would be good toward improving maintainability in the future.

Reply:
Definitely agree @stratease . 

This was a bit confusing for me. Even variable naming was off, imo. `$ticket_id` was ref to the Attendee instead of the Ticket, which was named `$product_id`... took me some time to break down all the logic.

My fix now is towards that direction: to use the methods from RSVP/Ticket classes, instead of updating a bunch of meta fields directly.

Also, [this conditional here](https://github.com/the-events-calendar/event-tickets/pull/3219/files#diff-98b75fdcf668fd80207d85d830c6fcb599c8bed942f16edcefed124f460ac54aR632), and all updates after it, should be centralized under `increase_rsvp()` or `increase_ticket()`

What do you think?